data/net/minecraft/world/level/pathfinder/Node.mapping

```diff
@@ -37,9 +37,14 @@ CLASS net/minecraft/world/level/pathfinder/Node
 	METHOD distanceToSqr (Lnet/minecraft/world/level/pathfinder/Node;)F
 		COMMENT Returns the squared distance to another path point
 		ARG 1 pathpoint
+	METHOD distanceToXZ (Lnet/minecraft/world/level/pathfinder/Node;)F
+		ARG 1 pathpoint
```

Comment:
I would rename this parameter (and all others in the class) to `point` as `pathpoint` is two words and should be camelCase anyways.